cluster/kubectl.sh

```diff
@@ -26,6 +26,10 @@ source ${KUBEVIRT_DIR}/hack/config.sh
 
 if [ "$1" == "console" ] || [ "$1" == "vnc" ]; then
     ${KUBEVIRT_DIR}/_out/cmd/virtctl/virtctl --kubeconfig=${kubeconfig} "$@"
+elif [ "$1" == "start" ] || [ "$1" == "stop" ]; then
```

Comment:
That is not needed anymore. You can simply us `cluster/virtctl.sh`. The other ifs are only there for backward-compatibility. Will add it to the docs too on #910.

Reply:
will revert back